### PR TITLE
Sfl 284

### PR DESCRIPTION
**Pull request by saketPandhare-Webonise**

### CODE REVIEW

File: SUNFLOWR/SUNFLOWR/ExploreVC/MoonCalendarDetail/MoonCalendarDetailVC.swift [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            self.setLblFontForHeader(toHeight: toHeight)
+            self.view.layoutIfNeeded()
+        }
+    }
+    
+    /// This function is used to set font to title label
+    /// - Parameter toHeight: height
+    func setLblFontForHeader(toHeight: CGFloat) {
+        if (toHeight == StickyHeader.HeightExpanded) {
+            //self.viewSeperator.isHidden = true
```
Comment (rakeshM-Webonise) [inline SUNFLOWR/SUNFLOWR/ExploreVC/MoonCalendarDetail/MoonCalendarDetailVC.swift:111]: remove commented code

File: SUNFLOWR/SUNFLOWR/ExploreVC/MoonCalendarDetail/MoonCalendarDetailVC.swift [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            self.lblNavigationTitle.font = UIFont(name: FontName.HB_BOLD, size: FontSize.TWENTYFIVE)!
+        }else {
+            // self.viewSeperator.isHidden = false
+            // self.viewSeperator.addShadowToStickyHeader()
```
Comment (rakeshM-Webonise) [inline SUNFLOWR/SUNFLOWR/ExploreVC/MoonCalendarDetail/MoonCalendarDetailVC.swift:115]: +1

File: SUNFLOWR/SUNFLOWR/ExploreVC/MoonCalendarDetail/MoonCalendarDetailVC.swift (hunk 1 above)
Comment (saketPandhare-Webonise) [reply SUNFLOWR/SUNFLOWR/ExploreVC/MoonCalendarDetail/MoonCalendarDetailVC.swift:111]: This is required here ,will remove in Next PR ,as we need to implement  this functionality 
Currently in testing

File: SUNFLOWR/SUNFLOWR/ExploreVC/MoonCalendarDetail/MoonCalendarDetailVC.swift (hunk 2 above)
Comment (saketPandhare-Webonise) [reply SUNFLOWR/SUNFLOWR/ExploreVC/MoonCalendarDetail/MoonCalendarDetailVC.swift:115]: This is required here ,will remove in Next PR ,as we need to implement  this functionality 
Currently in testing

**Pull request merged**